src/config.h.in

```diff
@@ -187,9 +171,6 @@
 /* Define to 1 if you have the <stdlib.h> header file. */
 #undef HAVE_STDLIB_H
 
-/* Define to 1 if you have the `strcasecmp' function. */
-#undef HAVE_STRCASECMP
```

Comment:
HAVE_STRCASECMP and HAVE_STRNCASECMP are checked in [strcasecmp.h](https://github.com/macports/macports-base/blob/master/src/pextlib1.0/strcasecmp.h#L39).

Reply:
I think the correct fix is to just remove that file (along with the implementation). am I wrong?1

Reply:
Looks like it.